code/modules/power/apc/apc_main.dm

```diff
@@ -302,9 +303,35 @@
 	)
 	return data
 
+/obj/machinery/power/apc/proc/connect_remote_access(mob/remote_user)
+	if(opened)
+		return
+	remote_control_user = remote_user
+	ui_interact(remote_user)
+	remote_user.log_message("remotely accessed [src].", LOG_GAME)
+	say("Remote access detected.[locked ? " Interface unlocked." : ""]")
+	to_chat(remote_control_user, span_danger("[icon2html(src, remote_control_user)] Connected to [src]."))
+	if(locked)
+		playsound(src, 'sound/machines/terminal_on.ogg', 25, FALSE)
+		locked = FALSE
+	playsound(src, 'sound/machines/terminal_alert.ogg', 50, FALSE)
```

Comment:
is it normal this will play 2 sounds at the exact same time?

Reply:
that's intended, kept from previous version